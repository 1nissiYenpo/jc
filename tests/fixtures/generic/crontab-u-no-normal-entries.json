{"variables":[],"schedule":[{"occurrence":"daily","user":"root","command":"/bin/sh do_the_thing"}]}
