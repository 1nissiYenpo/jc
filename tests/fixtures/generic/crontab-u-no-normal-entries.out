#this is a test for the jc module
@daily root /bin/sh do_the_thing
